.github/workflows/ios_release.yml

```diff
@@ -68,7 +72,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         submodules: recursive
-        ref: ${{ env.branch }}
+        ref: ${{ inputs.commit-sha || env.branch }}
```

Comment:
This change is largely transparent to the fastlane plugin because we’re just checking out a different ref (commit vs branch HEAD) and proceed from there.